core/src/templates/components/global-footer/global-footer.twig

```diff
@@ -11,14 +11,19 @@
 {% if template_path_logo is empty -%}
   {%- set template_path_logo = "@decanter/components/logo/logo.twig" -%}
 {%- endif %}
+{% if external_link_text is empty -%}
+  {%- set external_link_text -%}
+    <span class="su-global-footer__link-a11y">(link is external)</span>
+  {% endset %}
+{%- endif %}
 
 <div class="su-global-footer {{ modifier_class }}">
-  <div class="su-global-footer__container">
+  <div class="su-global-footer__container" title="Common Stanford resources"|t>
     <div class="su-global-footer__brand">
       {%- include template_path_logo -%}
     </div>
     <div class="su-global-footer__content">
-      <nav aria-label="global footer menu">
+      <nav aria-label="global footer menu" title="All links are external"|t>
```

Comment:
@sherakama and @yvonnetangsu,

I changed this up a bit based on both of your feedback. Thank you.


This says:
"Common Stanford resources. You are currently in a group."

Moving on it says:
"Global footer menu"
Then after a pause it says:
"Links are external. You are currently in a navigation"

Reply:
Also, on hover, if they don't already have a title, they inherit "Links are external"

Reply:
If you think this will fly for Stanford, I will take this to SOAP office hours on 9/1.

Reply:
@cjwest , instead of using "title", is it possible to write the aria-label as "Stanford global footer menu with all or mostly external links", or add a SR only H2 or H3 somewhere in the footer that says the same? Maybe could ask Level Access during the office hour.

Reply:
```suggestion
      <nav aria-label="global footer menu" title="All links are external">
```